dds_cli/user.py

```diff
@@ -312,40 +309,30 @@ def token_report(self, token_metadata=None):
         LOG.info(f"[{markup_color}]{sign}  {message} {sign} [/{markup_color}]")
         LOG.info(f"[{markup_color}]Token age: {readable_timedelta(age)}[/{markup_color}]")
 
-        if lifetime.total_seconds() < 0:
+        if age > dds_cli.TOKEN_MAX_AGE:
```

Comment:
I would prefer to go more lightly on relying on `dds_cli.TOKEN_MAX_AGE`, such that the lifetime is calculated once in `__token_dates()` and than only used throughout the conditions.

Reply:
If the common goal is to get rid of dds_cli.TOKEN_MAX_AGE, then let me know. I was about to delete it anyway. @inaod568 @alneberg

Reply:
Yes,  I think we can get rid of that one if we can read the expiration time directly from the token. I believe Matthias kept it as a backup in case the age wouldn't be readable for some reason.

Reply:
Yes, indeed that was the initial, nowadays obsolete consideration. When I first wrote that code, I still hat the issue with the _PyJWT Invalid Payload Padding_ exception being raised, so I assumed that reading the header might occasionally fail.